stdlib/unittest/mock.pyi

```diff
@@ -176,9 +175,12 @@ class _patch(Generic[_T]):
         kwargs: Mapping[str, Any],
     ) -> None: ...
     def copy(self) -> _patch[_T]: ...
+    @overload
+    def __call__(self, func: _TT) -> _TT: ...
+    @overload
     def __call__(self, func: Callable[..., _R]) -> Callable[..., _R]: ...
     def decorate_class(self, klass: _TT) -> _TT: ...
-    def decorate_callable(self, func: _F) -> _F: ...
+    def decorate_callable(self, func: Callable[..., _R]) -> Callable[..., _R]: ...
```

Comment:
It looks like you changed the wrong line.

Reply:
No I think the decorate_callable signature should be the same as its respective __call__ overload since __call__ is basically just delegating to it. This additional change ensures they stay consistent.